packages/appstore/src/components/transaction-list/transactions-for-a-day/__tests__/transactions-for-a-day.spec.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import TransactionsForADay from '../transactions-for-a-day';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import { render, screen } from '@testing-library/react';
+
+jest.mock('@deriv/hooks', () => ({
+    ...jest.requireActual('@deriv/hooks'),
+    useActiveWallet: jest.fn(() => ({
+        loginid: '12345678',
+    })),
+}));
+
+describe('TransactionsForADay', () => {
+    let mocked_props: Required<React.ComponentProps<typeof TransactionsForADay>>;
+
+    beforeEach(() => {
+        mocked_props = {
+            day: '01 Jan 2023',
+            transaction_list: [
+                {
+                    account_category: 'wallet',
+                    // @ts-expect-error we don't really need a full config here
+                    account_currency_config: {
+                        code: 'USD',
+                        type: 'fiat',
+                    },
+                    account_type: 'doughflow',
+                    action_type: 'deposit',
+                    amount: 75,
+                    balance_after: 100,
+                    is_virtual: 0,
+                    landing_company_shortcode: 'svg',
+                    transaction_id: 123456789,
+                },
+                {
+                    account_category: 'wallet',
+                    // @ts-expect-error we don't really need a full config here
+                    account_currency_config: {
+                        code: 'USD',
+                        type: 'fiat',
+                    },
+                    account_type: 'doughflow',
+                    action_type: 'transfer',
+                    amount: 50,
+                    balance_after: 200,
+                    is_virtual: 0,
+                    landing_company_shortcode: 'svg',
+                    to: {
+                        loginid: '1234',
+                    },
+                    from: {
+                        loginid: '12345678',
+                    },
+                    transaction_id: 987654321,
+                },
+            ],
+        };
+    });
+
+    const render_container = (mock_store_override = {}) => {
+        const mock_store = mockStore(mock_store_override);
```

Comment:
@rostislav-deriv Since you are using same mocked_props , why beforeEach is needed here?

Reply:
To reset the changes made to them in each test case, I was following the way it's being done in `.../wallet-transfer/wallet-transfer-tile/__tests__/wallet-transfer-tile.spec.tsx` and other places